### PR TITLE
Parseout non-oss repo on Power

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dvaleev@suse.com>

### CODE REVIEW

File: package/skelcd-control-openSUSE.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Mar 18 17:04:19 UTC 2015 - dvaleev@suse.com
+
+- Parse out non-oss repos for Power
```
Comment (jreidinger) [inline package/skelcd-control-openSUSE.changes]: any source why this change is needed?

Comment (k0da) [inline package/skelcd-control-openSUSE.changes]: https://openqa.opensuse.org/tests/52695/modules/yast2_lan/steps/6

Comment (jreidinger) [inline package/skelcd-control-openSUSE.changes]: ok, so please add it to changes something like `Parse out non-oss repos for Power as it is not build for it` and add comment to spec file, so we know that we can remove if we start building for ppc

**Comment by jreidinger** [comment]
LGTM

**Comment by roman-neuhauser** [comment]
does not lgtm to me:

* s/Parseout/Exclude/
* the reason needs to go into the commit message
* the xsl file could use some formatting: unnecessary empty lines, horribly long lines

**Comment by k0da** [comment]
@roman-neuhauser I addressed two out of three issues with my last commit.

**Comment by k0da** [comment]
Please merge
